release_dashboard/lib/widgets/common/dialog_prompt.dart

```diff
@@ -4,37 +4,113 @@
 
 import 'package:flutter/material.dart';
 
-/// Widget that prompts an alert dialogue to the current window and forces the user to choose between two options.
+/// Function that prompts an alert to the current window and forces the user to choose between two options.
 Future<String?> dialogPrompt({
   required BuildContext context,
-  required String title,
-  required String content,
-  required String leftOptionTitle,
-  required String rightOptionTitle,
-  VoidCallback? leftOptionCallback,
-  VoidCallback? rightOptionCallback,
+  required Widget title,
+  required Widget? content,
+  required String leftButtonTitle,
+  required String rightButtonTitle,
+  VoidCallback? leftButtonCallback,
```

Comment:
These should be `Future<VoidCallback>`

In UI, it's preferred to have everything async so you don't hold the rendering process

Reply:
Oh, that's a good idea.